Projects/b_u585i_iot02a_ntz/README.md

```diff
@@ -45,6 +46,13 @@ After importing the b_u585i_iot02a_ntz project into STM32CubeIDE, Build the proj
 
 To write the newly built image to flash, select the Run As button, then select the `Flash_ntz` target.
 
+### 3.1 Memory Usage
```

Comment:
What has been done to optimise the memory usage before publishing the numbers?  For example, I've do we know if all the allocated network buffers are being used?  Are we counting the static array used as a heap as consumed memory?  Etc.

Reply:
Thanks @RichardBarry. I didn't optimise the memory usage before this. I'll close this PR for now.